### PR TITLE
clean(feature-flag): FABRIC8_FEATURE_TOGGLES_API_URL use WIT_URL as p…

### PR DESCRIPTION
**Pull request by corinnekrych**
…roxy for prod, dev mode can point to service

Part of https://github.com/openshiftio/openshift.io/issues/2025

### CODE REVIEW

File: environments/openshift-prod-cluster.sh [hunk 1]
```diff
@@ -51,7 +51,8 @@ export FABRIC8_AUTH_API_URL=https://auth.openshift.io/api/
 export FABRIC8_FORGE_API_URL="https://forge.api.openshift.io"
 
 # this URL is used in dev mode only, for now feature-toggles.service.ts used WITH url as a proxy
```
Comment (jiekang) [inline environments/openshift-prod-cluster.sh]: While we're touching this file:

"used WITH url as a proxy"

Maybe meant to be:

"uses WIT url as a proxy"

?

File: environments/openshift-free-int-cluster.sh [hunk 2]
```diff
@@ -43,8 +43,9 @@ if [ -z "${FABRIC8_PIPELINES_NAMESPACE}" ]; then
   export FABRIC8_PIPELINES_NAMESPACE=""
 fi
 
-# this URL is used in dev mode only, for now feature-toggles.service.ts used WITH url as a proxy
```
Comment (jiekang) [inline environments/openshift-free-int-cluster.sh]: I'd like to keep this comment in the file

File: src/app/app.module.ts [hunk 3]
```diff
@@ -47,7 +47,6 @@ import {
 /*
  * Platform and Environment providers/directives/pipes
  */
-import { featureTogglesApiUrlProvider } from '../a-runtime-console/shared/feature-toggles.provider';
```
Comment (jiekang) [inline src/app/app.module.ts]: Can the file `a-runtime-console/shared/feature-toggles.provider` be removed from the repository as well? Or is it still being used elsewhere?

Comment (corinnekrych) [reply src/app/app.module.ts]: oops I miss to remove it, let me do it

**Comment by jiekang** [comment]
@corinnekrych I just noticed: You still need to provide `featureTogglesApiUrlProvider` in `app.module.ts`.

```
diff --git a/src/app/app.module.ts b/src/app/app.module.ts
index 84acad3d..59d7224e 100644
--- a/src/app/app.module.ts
+++ b/src/app/app.module.ts
@@ -59,7 +59,7 @@ import { AppState, InternalStateType } from './app.service';
 // Footer & Header
 import { FeatureBannerComponent } from './feature-flag/banner/feature-banner.component';
 import { FeatureFlagResolver } from './feature-flag/resolver/feature-flag.resolver';
-import { FeatureTogglesService } from './feature-flag/service/feature-toggles.service';
+import { featureTogglesApiUrlProvider, FeatureTogglesService } from './feature-flag/service/feature-toggles.service';
 import { FeatureWarningPageComponent } from './feature-flag/warning-page/feature-warning-page.component';
 
 import { FooterComponent } from './layout/footer/footer.component';
@@ -196,6 +196,7 @@ export type StoreType = {
     ErrorService,
     FeatureFlagResolver,
     FeatureTogglesService,
+    featureTogglesApiUrlProvider,
     Fabric8RuntimeConsoleResolver,
     Fabric8RuntimeConsoleService,
     {
```
will work. Right now the application doesn't load on Master.

**Comment by fabric8cd** [comment]
@corinnekrych fabric8/fabric8-ui:SNAPSHOT-PR-2492-1 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2492-fabric8-ui.badger.fabric8.io

**Pull request merged**